configs/baremetal_test.h

```diff
@@ -43,6 +43,9 @@
 /* Use Mbed TLS' timer implementation for Linux. */
 #define MBEDTLS_TIMING_C
 
+/* Needed for certificates in ssl_opt.sh */
+#define MBEDTLS_FS_IO
```

Comment:
The CI picked up some failures in `test_suite_pk` with `baremetal.h` + `baremetal_test.h`, which might be related to this (at least that's the only thing in the PR's diff that I can imagine could explain the failure, but maybe it's something else entirely): perhaps the failing tests are missing some requirements (such as `!MBEDTLS_USE_TINYCRYPT`) but this previously didn't show as they were skipped due to requiring file I/O anyway? That's just a wild guess, the failures need investigating to confirm.

Reply:
The culprit was PEM-encoded keys without having PEM-module defined. This was fixed by introducing the same keys as DER-encoded, and the server-key already had a DER-encoded counterpart.